includes/abstracts/abstract-wc-order.php

```diff
@@ -1668,9 +1668,11 @@ public function get_formatted_line_subtotal( $item, $tax_display = '' ) {
 	/**
 	 * Gets order total - formatted for display.
 	 *
+	 * @param string $tax_display Excl or incl tax display mode.
+	 *
 	 * @return string
 	 */
-	public function get_formatted_order_total() {
+	public function get_formatted_order_total( $tax_display ) {
```

Comment:
It's missing default value and `$display_refunded`,

This is how suppose to be:

```php
	/**
	 * Gets order total - formatted for display.
	 *
	 * @param string $tax_display      Type of tax display.
	 * @param bool   $display_refunded If should include refunded value.
	 *
	 * @return string
	 */
	public function get_formatted_order_total( $tax_display = '', $display_refunded = true ) {
```